docs/adrs/TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+# ADR 1: External Tooling
```

Comment:
Perhaps:

`# ADR [n]: [Title]`

...considering it is a template?

Reply:
Good call, thanks.